yeonjae/programmers-1/짝수와홀수.js

```diff
@@ -0,0 +1,11 @@
+function solution(num) {
+  let answer = "";
+
+  num % 2 ? (answer = "Odd") : (answer = "Even");
+  return answer;
+}
```

Comment:
```suggestion
function solution(num) {
  return num % 2 === 0 ? "Even" : "Odd";
}
```
- 바로 위에처럼 리턴해줄 수 있으며 나머지 연산할 때는 조건식을 명확하게 해주시는게 좋습니다 👍

Reply:
넵 확인했습니다 😃😃😃